website/docs/r/resource_tag.html.markdown

```diff
@@ -44,7 +44,7 @@ The `ibm_resource_tag` resource provides the following [Timeouts](https://www.te
 Review the argument references that you can specify for your resource.
 
 - `resource_id` - (Required, String) The CRN of the resource on which the tags is be attached.
-- `resource_type` - (Optional, String) The resource type on which the tags should be attached.
+- `resource_type` - (Optional, String) The resource type on which the tags should be attached. This is valid for SoftLayer (IMS) resources only. The resource_type allowed values are: `SoftLayer_Virtual_DedicatedHost`, `SoftLayer_Hardware`, `SoftLayer_Hardware_Server`, `SoftLayer_Network_Application_Delivery_Controller`, `SoftLayer_Network_Vlan`, `SoftLayer_Network_Vlan_Firewall`, `SoftLayer_Network_Component_Firewall`, `SoftLayer_Network_Firewall_Module_Context`, `SoftLayer_Virtual_Guest`. A wrong value would result in an error in the `terraform apply` command.
```

Comment:
Use `Classic Infrastructure` instead of `SoftLayer (IMS)`

Reply:
Fixed